### PR TITLE
Add missing error check when adding a magnet link to deluge

### PR DESCRIPTION
**Pull request by smmalis37**
#### Database Migration
NO

#### Description
Add missing error check when adding a magnet link to deluge. The same error check is already present for adding from a file, just not a magnet.

### CODE REVIEW

**Comment by markus101** [comment]
@Taloth I think you discussed this on IRC, what's the verdict?

**Comment by Taloth** [comment]
No verdict exactly. I simply stated that we first need to know why deluge would reject it and how, so we can properly mock DelugeProxy.
The suggested change doesn't 'hurt', so I'm ok to have it merged in, it's just incomplete imho.

**Comment by smmalis37** [comment]
Unfortunately I've been unable to reproduce the behavior this was meant to fix, so I'm unable to get logs with the details Taloth requested.

File: src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs [hunk 1]
```diff
@@ -35,6 +35,11 @@ protected override string AddFromMagnetLink(RemoteEpisode remoteEpisode, string
         {
             var actualHash = _proxy.AddTorrentFromMagnet(magnetLink, Settings);
 
+            if (actualHash.IsNullOrWhiteSpace())
+            {
+                throw new DownloadClientException("Deluge failed to add magnet " + magenetLink);
```
Comment (Thirrian) [inline src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs]: magenetLink (typo) 😃

Comment (smmalis37) [reply src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs]: That's what I get for making this change directly in github's ui...

**Pull request merged**